Logger/LogFactory.cs

```diff
@@ -1,11 +1,24 @@
-﻿namespace Logger
+﻿namespace Logger;
+
+public class LogFactory
 {
-    public class LogFactory
+    private string? _Path;
+
+    public BaseLogger CreateLogger(string className)
     {
-        public BaseLogger CreateLogger(string className)
+        if (_Path is null)
         {
-            
-            return null;
+            return null!;
```

Comment:
```suggestion
            return null;
```
Same comment here, avoid using `!`, the better solution would be to change your method signature on line 7 to return a nullable `BaseLogger`

So something like this:
`public BaseLogger? CreateLogger(string className)`

Reply:
Yes, that does seem to simplify things in the class, though I did have to add the operators to pieces of several tests as a result of modifying to your suggestion. Better in the tests than in the classes, I suppose.